prawns/tips/src/handlers/tips.tips_new.event.ts

```diff
@@ -46,13 +41,52 @@ export default (network: SubstrateNetwork) =>
       who: decodeAddress(who),
       finder: rootAccount,
       tipValue,
-      reason: hexToString(reason),
+      reason,
       deposit: (await getDeposit(apiAtBlock, newTipEvent.tipHash))?.toBigInt(),
     });
 
     await ctx.store.save(tipModel);
   };
 
+function getMultiSigCallArgs(ctx: EventHandlerContext, proxyCallArgs: any): {
+  who: string,
+  reason: string,
+  tipValue: bigint | null,
+} {
+  const c = new Codec(ctx._chain.description.types);
+  const data = c.decodeBinary(ctx._chain.description.call, proxyCallArgs);
+
+  const who = data.value?.who ? '0x' + Buffer.from(data.value.who).toString('hex') : null;
+  const reason = data.value?.reason ? Buffer.from(data.value.reason).toString() : null;
+  const tipValue = data.value?.tipValue ? Buffer.from(data.value.tipValue).toString() : null;
+
+  if (!who || !reason) {
+    throw new Error(`Could not decode multisig call for: ${data}`)
+  }
+
+  return {
+    who,
+    reason,
+    tipValue: tipValue ? BigInt(tipValue) : null,
+  };
+}
+
+function getArgsFromCall(ctx: EventHandlerContext, extrinsic: SubstrateExtrinsic, proxyCallArgs: any): {
+  who: string,
```

Comment:
NIT: Add an `interface`|`type` like
```
interface TipCallArgs {
  who: string,
  reason: string,
  tipValue: bigint | null,
}
```

Reply:
Yeah, I was so happy it's solved that I overlooked any improvement! Thanks 👍